### PR TITLE
feat: Add more static functions

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: src/connectors/AuthServerConnector.ts [hunk 1]
```diff
@@ -11,16 +11,16 @@ export class AuthServerConnector extends AbstractConnector {
   }
 
   activate = async (): Promise<ConnectorUpdate<string | number>> => {
-    const identity = this.provider.getIdentity()
+    const account = this.provider.getAccount()
```
Comment (fzavalia) [inline src/connectors/AuthServerConnector.ts:14]: To make the getIdentity function private, I can use the getAccount result to determine if the user has signed in already. Both the identity and the account are stored and deleted on the same flows, so the existence of one determines the existence of the other. (Unless one is manually deleted from storage)

File: src/providers/AuthServerProvider.ts [hunk 2]
```diff
@@ -127,6 +127,27 @@ export class AuthServerProvider {
     sso.localStorageStoreIdentity(signer, identity)
   }
 
+  /**
+   * Get the persisted account from local storage.
+   */
+  static getAccount = () => {
```
Comment (fzavalia) [inline src/providers/AuthServerProvider.ts:133]: Used by the explorer website to determine if the user is already connected and render the flow accordingly

File: src/providers/AuthServerProvider.ts [hunk 3]
```diff
@@ -127,6 +127,27 @@ export class AuthServerProvider {
...
+    return localStorage.getItem(STORAGE_KEY_ADDRESS)
+  }
+
+  /**
+   * Clears the identity and the rest of the persisted data created by this provider.
+   */
+  static deactivate = () => {
```
Comment (fzavalia) [inline src/providers/AuthServerProvider.ts:140]: Used by the explorer website to clear data if the user decides to change accounts.

**Review by juanmahidalgo — APPROVED** [review]
looks good!! 👍

**Pull request merged**